packages/instant-apps-components/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "homepage": "https://esri.github.io/instant-apps-components",
   "name": "@esri/instant-apps-components",
-  "version": "1.0.0-beta.129",
+  "version": "1.0.0-beta.130",
```

Comment:
increment version to `1.0.0-beta.131` since `1.0.0-beta.130` was already published